modules/contact/include/problems/AugmentedLagrangianContactProblem.h

```diff
@@ -44,7 +44,15 @@ class AugmentedLagrangianContactProblem : public ReferenceResidualProblem
                             const Real ref_resid,
                             const Real div_threshold) override;
 
+  const unsigned int & getNumLagrangianUpdateIterations() const
+  {
+    return _num_lagrangian_update_iterations;
+  }
+
 private:
-  int _num_lagmul_iterations;
-  int _max_lagmul_iters;
+  /// maximum mumber of augmented lagrange iterations
+  const unsigned int _maximum_lagrangian_update_iterations;
+
+  /// current augmented lagrange iteration number
+  unsigned int _num_lagrangian_update_iterations;
```

Comment:
I have suggestions for name changes:

_maximum_lagrangian_update_iterations -> _maximum_number_lagrangian_iterations

_num_lagrangian_update_iterations -> _lagrangian_iteration_number